bpf/aya-bpf/src/programs/lsm.rs

```diff
@@ -10,6 +10,50 @@ impl LsmContext {
     pub fn new(ctx: *mut c_void) -> LsmContext {
         LsmContext { ctx }
     }
+
+    /// Coerces a `T` from the `n`th argument to the LSM hook where `n` starts at 0 and
```

Comment:
I think this is a bit too technical, I'd rephrase it as:

```
/// Returns the `n`th argument passed to the LSM hook, starting from 0.
```

Reply:
Sure